### DISCUSSION TITLE
Working NerdMiner with minor Coins

### DISCUSSION THREAD

**Discussion by fabioce** [BitMaker-hub/NerdMiner_v2#527] [Ideas]
What about to customize NerdMiner to work with others SHA256 more simple coins?
Like BCH,BSV,DGB.
I think is simple to create customization

**Comment by iijamesy** [comment]
I have 5 nerdminers currently running on my Bch pool no issue. 
i have been playing around with the NM firmware to try and get the screen to support bch. so i can view bch price and current bch block just like it shows for BTC. but as of right now i haven't had any luck on the nerdminer firmware end. But i am also completely new to code, firmware etc. still figuring everything out.

**Reply by pashol** [reply]
You'd need to change src/monitor.h and some lines in monitor.cpp.

Without knowing too much about the subject, you'd need a reliable provider of the data in a structured way.

**Reply by hugohfsouza** [reply]
Besides unmineable, is there any other public pool?

**Reply by iijamesy** [reply]
> Besides unmineable, is there any other public pool?

Not many pools support low hashrate.
You'll have to check a ton of different pools to see if your low hashrate is supported.

My pool is now public and accepts low hash rate. It's for BCH. Pool.bchsolo.com use port 3333 and make sure you use your bch address on the nerdminer.

Unmineable is a good option.
But there is others like public-pool.io and pool.nerdminers.org that will work as well.

**Reply by gungriffonbr** [reply]
> > Besides unmineable, is there any other public pool?
> 
> Not many pools support low hashrate. You'll have to check a ton of different pools to see if your low hashrate is supported.
> 
> My pool is now public and accepts low hash rate. It's for BCH. Pool.bchsolo.com use port 3333 and make sure you use your bch address on the nerdminer.
> 
> Unmineable is a good option. But there is others like public-pool.io and pool.nerdminers.org that will work as well.

on unmineable they raised the difficulty dont work anymore

**Reply by CreddeDK** [reply]
> > > Besides unmineable, is there any other public pool?
> > 
> > 
> > Not many pools support low hashrate. You'll have to check a ton of different pools to see if your low hashrate is supported.
> > My pool is now public and accepts low hash rate. It's for BCH. Pool.bchsolo.com use port 3333 and make sure you use your bch address on the nerdminer.
> > Unmineable is a good option. But there is others like public-pool.io and pool.nerdminers.org that will work as well.
> 
> on unmineable they raised the difficulty dont work anymore

True, even though that in late January there was a newsflash on unmineable:

"SHA256 miners: We’ve recently pushed an update to restore support for low hash rate devices (all ports) and improve pool compatibility. If you’re still experiencing connection issues, be sure to contact us via our support email. We’ll do our best to ensure your device is supported. Happy mining!"

So, even with that "update" I cannot get it to work. Too bad as it was working quite well before.

**Reply by Cr4zyT1zy** [reply]
_Besides unmineable, is there any other public pool?_

Just to sperimentation.

You can try to connect at https://blockminerz.com/ to extrct digibyte with Sha256 algo.

Use in:
pool : eu-stratum.blockminerz.com
port : 3311
look into web site to chose your location.

BTC address in nerminer menu put your Digibyte address.name_of_worker
Password you can define the difficult of oyur miner. I use d=0.05

Then to read the information from the NerdMiner i use o windows tool in demo version.
Its is freeware at this Url: https://www.ltsoftware.it/NerdViewer.html

**Reply by andrewjswan** [reply]
I try https://unmineable.com/?algorithm=sha256 on 3333 port, but I received a message in the log that it is `not supported`

Try https://dgb-sha.solopool.org/help but hashrate 0 ... It seems like there isn't enough speed.

**Comment by m8ax** [comment]
probe this firmware https://m8ax.github.io/MvIiIaX-NerdMiner_V2-DeV/
